src/components/Composer.vue

```diff
@@ -1008,10 +1002,9 @@ export default {
 			return `${alias.name} <${alias.emailAddress}>`
 		},
 		async discardDraft() {
-			this.state = STATES.DISCARDING
 			const id = await this.draftsPromise
 			await this.$store.dispatch('deleteMessage', { id })
```

Comment:
Please add a try catch around this dispatch and call `showError(...)` in the catch block. Otherwise, the user won't be notified in case of failure.

Reply:
yep. fixed the try..catch, i tested it with new message and it works, but discarding while in an outbox message, it doesnt work. Will fix it in this pr